### DISCUSSION TITLE
Auto sort projects based on their activity

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [schmidfelix/timey#12] [Ideas] [answered]
It would be cool to have an option to auto sort projects based on their activity

**Comment by schmidfelix** [comment, marked as answer]
Hey Sebastian,

Thanks for the suggestion - I really like the idea, however I see the following problem when taking a look at #10: 

When you can either sort by project name, the last activity and somehow reorder them manually, I can imagine that this will go a little crazy.

Since I like the idea of adding multiple sort-by conditions a little more than reordering them manually, I think I'm going to implement that for now and maybe take a look at manually reordering them at some later point.

Or would you prefer a manual reordering more than sort by name/last activity/etc.?

**Reply by SebastiaanKloos** [reply]
Nope automatically sort (by name, activity) is much more needed than the manual sort I think👍
